rasa_sdk/forms.py

```diff
@@ -656,46 +557,41 @@ async def run(
     def __str__(self) -> Text:
         return f"FormAction('{self.name()}')"
 
-    def _get_entity_type_of_slot_to_fill(
-        self,
-        slot_to_fill: Optional[Text],
-    ) -> Optional[Text]:
-        if not slot_to_fill:
-            return None
-
-        mappings = self.get_mappings_for_slot(slot_to_fill)
-        mappings = [m for m in mappings if m.get("type") == "from_entity"]
-
-        if not mappings:
-            return None
-
-        entity_type = mappings[0].get("entity")
-
-        for i in range(1, len(mappings)):
-            if entity_type != mappings[i].get("entity"):
-                return None
 
-        return entity_type
+class ValidationAction(Action, ABC):
+    """A helper class for slot validations and extractions of custom slots."""
 
+    def name(self) -> Text:
+        """Unique identifier of this simple action."""
+        if self.global_slot_mappings:
+            return VALIDATE_GLOBAL_SLOT_MAPPINGS_NAME
 
-class FormValidationAction(Action, ABC):
-    """A helper class for slot validations and extractions of custom slots."""
+        raise NotImplementedError("An action must implement a name")
 
     def form_name(self) -> Text:
         """Returns the form's name."""
         return self.name().replace("validate_", "", 1)
 
+    @property
+    def global_slot_mappings(self) -> bool:
```

Comment:
Would this method have to be overriden by the user in case it can be used outside forms - just to make sure I account for this in the docs?